### DISCUSSION TITLE
issur related to sending email due to Gmail_API insufficient authentication scopes problem

### DISCUSSION THREAD

**Discussion by lenin12345-dev** [emailjs-com/help#103] [Q&A]
POST https://api.emailjs.com/api/v1.0/email/(Precondition Failed)
Contact.jsx:68 
EmailJSResponseStatus {status: 412, text: 'Gmail_API: Request had insufficient authentication scopes.'}
status
: 
412
text
: 
"Gmail_API: Request had insufficient authentication scopes."

can any one give some idea to resolve this?

**Comment by xr0master** [comment]
To solve this problem, you need to go to EmailJS dashboard -> Services, click on the Gmail service in your list, and reconnect your account (click on the Edit button, disconnect, and then connect again).
On the popped-up screen, press the "Allow" button (see below) to allow us to send the emails through your Gmail account.
